WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsItem.swift

```diff
@@ -0,0 +1,201 @@
+                        .headlineStyle()
+                    Spacer()
+                    Image(systemName: "exclamationmark.circle")
+                        .foregroundColor(.withColorStudio(name: .red, shade: .shade60))
+                        .renderedIf(viewModel.informationIsMissing)
+                }
+
+                VStack(alignment: .leading, spacing: Layout.collapsibleViewBottomLabelVerticalSpacing) {
+                    HStack {
+                        Text(viewModel.description)
+                        Spacer()
+                        Text(viewModel.hsTariffNumber)
+                    }
+                    HStack {
+                        Text(viewModel.originCountry.name)
+                        Spacer()
+                        Text(viewModel.weightPerUnit)
+                        Text("•")
+                        Text(viewModel.valuePerUnit)
+                    }
+                }.renderedIf(isCollapsed)
+                    .foregroundColor(.primary)
+                    .subheadlineStyle()
+                    .padding(.trailing, Layout.collapsibleViewBottomContentTrailingPadding)
+            }
+            .padding(.top, Layout.collapsibleViewTopPadding)
+        }, content: {
+            VStack(alignment: .leading, spacing: Layout.collapsibleViewVerticalSpacing) {
+                Divider()
+                HStack {
+                    Text(Localization.descriptionTitle)
+                        .foregroundColor(.primary)
+                        .subheadlineStyle()
+                    Spacer()
+                    Image(systemName: "info.circle")
+                        .foregroundColor(Color(.wooCommercePurple(.shade60)))
+                }
+                .padding(.top, Layout.descriptionTopPadding)
+                TextField("", text: $viewModel.description)
+                    .padding(Layout.extraPadding)
+                    .roundedBorder(cornerRadius: Layout.borderCornerRadius, lineColor: Color(.separator), lineWidth: Layout.borderLineWidth)
+                Text(Localization.HSTarriffNumber)
+                    .foregroundColor(.primary)
+                    .subheadlineStyle()
+                TextField(Localization.HSTarriffNumberPlaceholder, text: $viewModel.hsTariffNumber)
+                    .padding(Layout.extraPadding)
+                    .roundedBorder(cornerRadius: Layout.borderCornerRadius, lineColor: Color(.separator), lineWidth: Layout.borderLineWidth)
+                Button {
+                    isShowingHSTarrifInfoWebView = true
+                } label: {
+                    HStack(alignment: .top, spacing: Layout.hsTarriffNumberMoreInfoVerticalSpacing) {
+                        Image(systemName: "info.circle")
+                        Text(Localization.HSTarriffNumberMoreInfo)
+                    }
+                    .foregroundColor(Color(.wooCommercePurple(.shade60)))
+                    .footnoteStyle()
+                }
+
+                HStack {
```

Comment:
I think we should top-align this `HStack` so they remain aligned even if one of them is missing a value:

<img width="307" alt="Screenshot 2024-12-30 at 16 38 10" src="https://github.com/user-attachments/assets/cf091b65-a528-4e04-9575-dad4b82443f8" />

Reply:
Good point, done in 4ae10d718a6271733e67c3431bc672266bcbb952